web_src/js/utils/dom.js

```diff
@@ -51,8 +51,22 @@ export function isElemHidden(el) {
   return res[0];
 }
 
-export function queryElemSiblings(el, selector = '*') {
-  return Array.from(el.parentNode.children).filter((child) => child !== el && child.matches(selector));
+function applyElemsCallback(elems, fn) {
+  if (fn) {
+    for (const el of elems) {
+      fn.call(el, el); // call the function with the element as "this", and pass the element as the first argument
+    }
```

Comment:
```suggestion
    for (const el of elems) {
      fn(el);
    }
```

`this` makes it easier to migrate existing jQuery code to the new framework.

But if you insist to remove `this` support, feel free to change it.

Reply:
Just use the first argument, all `this` usage in such mapping functions should be forbidden in new code.